ui/app/components/auth-form.js

```diff
@@ -326,17 +251,13 @@ export default Component.extend(DEFAULTS, {
           data.path = 'okta';
         }
       }
-      return this.authenticate.unlinked().perform(backend.type, data);
+      return this.handleAuth.unlinked().perform(backend.type, data);
```

Comment:
Testing tasks can be challenging, so may want to update this to an action with the args and then the parent action calls `perform()`

Reply:
Totally agree, it's hard for components to guarantee that the passed callback will be a task so I think it makes sense to update this to call some action with the args instead of invoking the task directly